content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md

```diff
@@ -465,8 +465,7 @@ There are some implicit conventions worth noting here:
 - The scheduler bypasses any nodes that don't have any `topologySpreadConstraints[*].topologyKey`
   present. This implies that:
 
-  1. any Pods located on those bypassed nodes do not impact `maxSkew` calculation - in the
-     above example, suppose the node `node1` does not have a label "zone", then the 2 Pods will
+  1. any Pods located on those bypassed nodes do not impact `maxSkew` calculation - [in the above example](/#example-conflicting-topologyspreadconstraints), suppose the node `node1` does not have a label "zone", then the 2 Pods will
```

Comment:
I believe the added hyperlink is not accurate. I have provided the suggestion below. 
Additionally, it would be great to address the failing build checks in order to make progress on this.

```suggestion
  1. any Pods located on those bypassed nodes do not impact `maxSkew` calculation - [in the above example](#example-conflicting-topologyspreadconstraints), suppose the node `node1` does not have a label "zone", then the 2 Pods will
```

Reply:
@dipesh-rawat Thank you for the suggestion